jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/DefaultControllerEnricherCreateTest.java

```diff
@@ -67,108 +73,65 @@ public void create_inKubernetesNoType_shouldCreateDeployment() {
         .isInstanceOf(Deployment.class);
   }
 
+  // TODO: Do we really need this comment?
```

Comment:
The comment needs to be moved to the relevant argument entry

Reply:
It explains why despite using DeploymentConfig as controller type we end up with a Deployment, so yes, it's necessary